WtWSMS/credits.txt

```diff
@@ -11,11 +11,11 @@ Loup99
 Symmachus
 
 $TITLE_OTHER_MODS_PROVIDING_CONTENT$
-Vertimanus for the "Community Culture Utility"
-Successions Expanded Team
 Chalk and Rootel for "More Faith Icons"
-herkles (aka Vyshan) for Dharmachakra
 Elder Kings 2 Mod Team for their sub-bookmark system
```

Comment:
Do we still need those given the vanilla change?

Reply:
Not sure what you mean? This overwrites the Vanilla credits with the actual mod credits.

Reply:
I'm referring to the credits to the Elder Kings 2 Mod Team for their sub-bookmark system. Does the new vanilla bookmark not overwrite it?

Reply:
Oh, right, yes. I'll remove that then.